x-pack/plugins/code/public/components/main/main.scss

```diff
@@ -271,3 +271,11 @@
 .code-no-min-width {
   min-width: unset !important;
 }
+
+.code-commit-id {
+  @include euiCodeFont;
```

Comment:
@daveyholler is this what you mean? please take a second look

Reply:
Perfect.